java/src/main/native/ai_onnxruntime_OnnxRuntime.c

```diff
@@ -16,7 +16,19 @@ JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OnnxRuntime_initialiseAPIBase(JNIEnv
                                                                           jint apiVersion) {
   (void)jniEnv; (void)clazz;  // required JNI parameters not needed by functions which don't call back into Java.
   const OrtApi* ortPtr = OrtGetApiBase()->GetApi((uint32_t)apiVersion);
-  return (jlong)ortPtr;
+  return (jlong) ortPtr;
+}
+/*
+ * Class:     ai_onnxruntime_OnnxRuntime
+ * Method:    initialiseTrainingAPIBase
+ * Signature: (JI)J
+ */
+JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OnnxRuntime_initialiseTrainingAPIBase
+  (JNIEnv * jniEnv, jclass clazz, jlong apiHandle, jint trainingApiVersion) {
+  (void)jniEnv; (void)clazz;  // required JNI parameters not needed by functions which don't call back into Java.
```

Comment:
>/ required JNI parameters not needed by functions which don't call back into Java. [](http://example.com/codeflow?start=31&length=82)

Suggest commenting out parameter names leaving only type, that would like nicer. JNIEnv* /* jniEnv */,

Reply:
This is the idiom I've used everywhere else in the JNI code, do you want it changed everywhere?

Reply:
It would be nice, less code and more readability. Feel free to address it in a separate PR.

Reply:
At your discretion.